apps/templates/content_templates.py

```diff
@@ -192,8 +191,27 @@ class ContentTemplatesResource(Resource):
 
 class ContentTemplatesService(BaseService):
 
+    def get(self, req, lookup):
+        active_user = g.get('user', {})
+        privileges = active_user.get('active_privileges', {})
+        results = super().get(req, lookup)
```

Comment:
imo it would be better to modify the lookup if needed, not sure if pagination would work otherwise.
shoudl be fine to add if needed:
```python
lookup.update({'$or': [{'is_public': True}, {'user': active_user['_id']}})
if privileges.get('personal_template'):
    lookup['$or'].append({'is_public': False})
```
not sure if it's all covered with that, just for illustration